### DISCUSSION TITLE
Ti* features for S3?

### DISCUSSION THREAD

**Discussion by mindless-bureaucrat** [developmentseed/tipg#134] [Ideas]
Hey Vincent & Crew,

Curious if there's anything out there already (or maybe in the works) for a TiPG -like project that provides OGC features through FastAPI, access but for blob storage. Think of a giant geoparqet or flatgeobuf in cloud storage and being able to query features from it. Similar to how with TiTiler we can point the query to a specific URL the server has access to. Just want to check before I (hopefully won't) roll-my-own.

**Comment by vincentsarago** [comment]
👋 @mindless-bureaucrat 

I've been thinking about making `backends` for tipg for a while, but didn't had time yet nor real use case (at least internally). 

I feel this could just be a documentation add instead of a proper refactor for TiPg (because then the name won't mean anything 😅 )
